pkg/cmd/server/origin/rest/storage_options.go

```diff
@@ -45,6 +46,35 @@ func StorageOptions(options configapi.MasterConfig) restoptions.Getter {
 			{Resource: "netnamespaces"}:                                        "registry/sdnnetnamespaces",
 			{Resource: "netnamespaces", Group: "network.openshift.io"}:         "registry/sdnnetnamespaces",
 		},
+		// storage versions:
+		[]schema.GroupVersionResource{
+			{"authorization.openshift.io", "v1", "clusterpolicybindings"},
+			{"authorization.openshift.io", "v1", "clusterpolicies"},
+			{"authorization.openshift.io", "v1", "policybindings"},
+			{"authorization.openshift.io", "v1", "rolebindingrestrictions"},
+			{"authorization.openshift.io", "v1", "policies"},
+			{"build.openshift.io", "v1", "builds"},
+			{"build.openshift.io", "v1", "buildconfigs"},
+			{"apps.openshift.io", "v1", "deploymentconfigs"},
+			{"image.openshift.io", "v1", "imagestreams"},
+			{"image.openshift.io", "v1", "images"},
+			{"oauth.openshift.io", "v1", "oauthclientauthorizations"},
+			{"oauth.openshift.io", "v1", "oauthaccesstokens"},
+			{"oauth.openshift.io", "v1", "oauthauthorizetokens"},
+			{"oauth.openshift.io", "v1", "oauthclients"},
+			{"project.openshift.io", "v1", "projects"},
+			{"quota.openshift.io", "v1", "clusterresourcequotas"},
+			{"route.openshift.io", "v1", "routes"},
+			{"network.openshift.io", "v1", "netnamespaces"},
+			{"network.openshift.io", "v1", "hostsubnets"},
+			{"network.openshift.io", "v1", "clusternetworks"},
+			{"network.openshift.io", "v1", "egressnetworkpolicies"},
+			{"template.openshift.io", "v1", "templates"},
+			{"user.openshift.io", "v1", "groups"},
+			{"user.openshift.io", "v1", "users"},
+			{"user.openshift.io", "v1", "identities"},
+		},
```

Comment:
Compare: https://github.com/openshift/origin/pull/14018

The idea is that we start with the existing resources as legacy GVRs in the core group. Every new resource will be stored in the apigroup GV directly.